samples/02_power_users_developers/building_a_change_detection_app_using_jupyter_dashboard.ipynb

```diff
@@ -102,7 +102,7 @@
     "gis = GIS()\n",
```

Comment:
<p>tags: <a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Landsat%20on%20AWS%22" target="_blank">Landsat on AWS</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Landsat%22" target="_blank">Landsat</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Multispectral%22" target="_blank">Multispectral</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22Multitemporal%22" target="_blank">Multitemporal</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22imagery%22" target="_blank">imagery</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22landsat%208%22" target="_blank">landsat 8</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22temporal%22" target="_blank">temporal</a>,&nbsp;<a href="https://www.arcgis.com/home/search.html?t=content&amp;q=tags:%22MS%22" target="_blank">MS</a></p><p>adding tags in search will make the search more robust</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/770/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>